pkgs/development/python-modules/cpe/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, fetchFromGitHub, buildPythonPackage, pytest, sphinx, setuptools }:
+
+buildPythonPackage rec {
+  pname = "cpe";
+  version = "1.2.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "nilp0inter";
+    repo = "cpe";
+    rev = "v${version}";
+    hash = "sha256-1hTOMbsL1089/yPZbAIs5OgjtEzCBlFv2hGi+u4hV/k=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [ sphinx pytest ];
+
+  pythonImportsCheck = [ "cpe" ];
+
+  # no test
```

Comment:
The upstream repo contains [tests](https://github.com/nilp0inter/cpe/tree/develop/tests).